prisma/schema.prisma

```diff
@@ -22,26 +22,21 @@ model Book {
   locationId      String
   libraryMemberId String?
   Checkout        Checkout[]
+  dateCreated     DateTime   @default(now())
 
   @@index([locationId], map: "Book_locationId_fkey")
 }
 
 model Genre {
-  id     String  @id @default(cuid())
-  name   String
-  bookId String?
-  Book   Book?   @relation(fields: [bookId], references: [id])
-
-  @@index([bookId], map: "Genre_bookId_fkey")
+  id   String @id @default(cuid())
```

Comment:
Schema change to prevent existing Genres and Regions from being deleted and recreated.